plugins/db-mongo/src/static/docker-compose/docker-compose.db.yml

```diff
@@ -1,6 +1,6 @@
 version: "3.7"
 services:
-  mongodb_container:
+  db:
```

Comment:
I am pretty sure that we also use' db' for MySQL and Postgres, so let's be consistent and change it for these DB plugins (not on this PR, just mentioning...)